editor.planx.uk/src/pages/FlowEditor/index.tsx

```diff
@@ -99,14 +104,88 @@ export const LastEdited = () => {
       <Typography variant="body2" fontSize="small">
         {message}
       </Typography>
-      {hasFeatureFlag("UNDO") && formattedOps && (
-        <Typography component="ul" pl={2}>
-          {[...new Set(formattedOps)].map((op, i) => (
-            <Typography variant="body2" fontSize="small" component="li" key={i}>
-              {op}
-            </Typography>
-          ))}
-        </Typography>
+    </Box>
+  );
+};
+
+export const EditHistory = () => {
```

Comment:
This should probably move to its own file now 👍

Reply:
Yes I agree - I've tried this twice - creating an `EditHistory.tsx` at the same level of `PreviewBrowser.tsx` but consistently getting an opaque loading error on the whole page when trying to import the component from there. Maybe our render bug again?

Don't want to lose more time to that right now before actual functionality is in place, this sholud be easy enough to come back to & tidy.